chicken-sloths/bangazon-api-sprint1
# Faker fixes

# Description
- Adds primary keys to Departments, Employees, Computers, Training_Programs
- Makes sure all employees have a unique computer
- Removes `employee_id` from Computers (fixes #69, fixes #70)
- Add primary key, `supervisor_id` to `Departments`

## Steps to Test Solution

1. `npm run data:generate`
1. `npm run db:generate`
1. Make sure `Departments` has a defined PK and `supervisor_id`
1. Look for primary keys on everything
